frontend/src/components/IstioStatus/IstioStatus.tsx

```diff
@@ -94,6 +100,10 @@ export class IstioStatus extends React.Component<Props> {
     let addonUnhealthy: boolean = false;
     let notReady: boolean = false;
 
+    if (this.props.istioAPIEnabled === false) {
+      return PFColors.Warning
+    }
```

Comment:
I wonder if this should really be a permanent warning if users have purposefully disabled istiod access by setting the config option?